cmd/common.go

```diff
@@ -111,3 +134,28 @@ func MakeDirs(devPath string) error {
 	}
 	return nil
 }
+
+func Die(message string) { 
+  fmt.Printf(message)
+  os.Exit(1)
+}
+
+func GolangHolochainDir(subPath ...string) string {
+  joinable := append([]string{os.Getenv("GOPATH"), "src/github.com/metacurrency/holochain", }, subPath...)
```

Comment:
the point of using filepath.join (which I'm glad you started because I wasn't doing it before) is to make things work cross-platform re "/" "\" filepath separators, so I think it really doesn't make too much sense to have the long string there with the "/" in it.  Looks like I had it that way in the code you updated, so my bad too!